builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -439,48 +447,48 @@ function store.get_formspec(dlgdata)
 
 		-- buttons
 		local description_width = W - 0.375*5 - 1 - 2*1.5
+		local left_base = "image_button[-1.55,0;0.7,0.7;" .. core.formspec_escape(defaulttexturedir)
 		formspec[#formspec + 1] = "container["
 		formspec[#formspec + 1] = W - 0.375*2
 		formspec[#formspec + 1] = ",0.1]"
 
 		if package.downloading then
-			formspec[#formspec + 1] = "button[-3.5,0;2,0.8;status;"
-			formspec[#formspec + 1] = fgettext("Downloading...")
-			formspec[#formspec + 1] = "]"
+			formspec[#formspec + 1] = "animated_image[-1.7,-0.15;1,1;downloading;"
+			formspec[#formspec + 1] = core.formspec_escape(defaulttexturedir)
+			formspec[#formspec + 1] = "cdb_downloading.png;3;400;]"
 		elseif package.queued then
-			formspec[#formspec + 1] = "button[-3.5,0;2,0.8;status;"
-			formspec[#formspec + 1] = fgettext("Queued")
-			formspec[#formspec + 1] = "]"
+			formspec[#formspec + 1] = left_base
+			formspec[#formspec + 1] = core.formspec_escape(defaulttexturedir)
+			formspec[#formspec + 1] = "cdb_queued.png;queued]"
 		elseif not package.path then
-			formspec[#formspec + 1] = "button[-3,0;1.5,0.8;install_"
-			formspec[#formspec + 1] = tostring(i)
-			formspec[#formspec + 1] = ";"
-			formspec[#formspec + 1] = fgettext("Install")
-			formspec[#formspec + 1] = "]"
+			local elem_name = "install_" .. i .. ";"
+			formspec[#formspec + 1] = "style[" .. elem_name .. "bgcolor=#71aa34]"
+			formspec[#formspec + 1] = left_base .. "cdb_add.png;" .. elem_name .. "]"
+			formspec[#formspec + 1] = "tooltip[" .. elem_name .. fgettext("Install") ..
```

Comment:
Bad translation. You should only ever do `..` with a translation if you're concatenating _complete sentences_. This isn't the case, so you should do this instead:

```lua
fgettext("Install $1", package.title) .. tooltip_colors
```

Reply:
I created tooltips because I didn't want to eliminate already existing strings. In this way, though, it means creating new strings and I guess you don't want to mess up with new strings so.. is better if I just get rid of the tooltips?

Reply:
Well, you could remove the package.title from the tooltip. But I don't think creating new strings is _too_ much of a problem

Reply:
I'm gonna play it safe and I'll keep the same string